latest-issue/src/main/java/net/androidweekly/latestissue/LatestIssueFragment.kt

```diff
@@ -52,7 +53,10 @@ class LatestIssueFragment : BaseFragment() {
         initViews()
         initListeners()
         if (arguments?.containsKey(ARG_ITEMS) == true) {
-            viewModel.setItems(arguments?.getParcelableArray(ARG_ITEMS))
+            arguments?.getParcelable<Issue>(ARG_ITEMS)?.let { viewModel.setItems(it) }
```

Comment:
This change will break down `View All` button in the latest issue screen, you should pass array of items not issue